package.json

```diff
@@ -40,7 +40,8 @@
     "eject": "react-scripts eject",
     "localdev": "scripts/localdev.sh",
     "predeploy": "npm run build",
-    "deploy": "gh-pages -a -d build"
+    "deploy": "gh-pages -a -d build",
+    "postinstall": "npm update && npm install solr-faceted-search-react"
```

Comment:
This is to make sure that we are getting the most recent version of the `solr-faceted-search-react`. Sometimes it gets the old version and it can cause errors.